### ISSUE TITLE
Warning when running build:custom:compose on PHP 7.4

### ISSUE THREAD

**Issue by magento-engcom-team** [magento/magento-cloud-docker#256]
Warning occurs when running **ece-docker build:custom:compose** on PHP 7.4.  The docker-compose.yml looked like it was created okay.

 
```java
 Deprecated: Unparenthesized `a ? b : c ? d : e` is deprecated. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /app/vendor/magento/magento-cloud-docker/src/Compose/ProductionBuilder.php on line 232```

**Comment by magento-engcom-team** [comment]
The issue was exported from the internal JIRA. The link to the original JIRA issue: https://jira.corp.magento.com/browse/MCLOUD-6393

**Comment by shiftedreality** [comment]
Fixed as part of #246
